spec.html

```diff
@@ -28884,7 +28904,45 @@ <h1>
                     1. If _vnDefinable_ is *false*, throw a *TypeError* exception.
                   1. If _declaredVarNames_ does not contain _vn_, then
                     1. Append _vn_ to _declaredVarNames_.
-          1. [id="step-evaldeclarationinstantiation-web-compat-insertion-point"] NOTE: Annex <emu-xref href="#sec-web-compat-evaldeclarationinstantiation"></emu-xref> adds additional steps at this point.
+          1. [id="step-evaldeclarationinstantiation-web-compat-insertion-point", normative-optional] If the host is a web browser or otherwise supports <emu-xref href="#sec-block-level-function-declarations-web-legacy-compatibility-semantics" title></emu-xref> and _strict_ is *false*, then
+            1. Let _declaredFunctionOrVarNames_ be the list-concatenation of _declaredFunctionNames_ and _declaredVarNames_.
+            1. For each |FunctionDeclaration| _f_ that is directly contained in the |StatementList| of a |Block|, |CaseClause|, or |DefaultClause| Contained within _body_, do
+              1. Let _F_ be StringValue of the |BindingIdentifier| of _f_.
+              1. If replacing the |FunctionDeclaration| _f_ with a |VariableStatement| that has _F_ as a |BindingIdentifier| would not produce any Early Errors for _body_, then
+                1. Let _bindingExists_ be *false*.
+                1. Let _thisEnv_ be _lexEnv_.
+                1. Assert: The following loop will terminate.
+                1. Repeat, while _thisEnv_ is not _varEnv_,
+                  1. If _thisEnv_ is not an Object Environment Record, then
+                    1. If ! _thisEnv_.HasBinding(_F_) is *true*, then
+                      1. [id="step-evaldeclarationinstantiation-web-compat-bindingexists", normative-optional] If the host is a web browser or otherwise supports <emu-xref href="#sec-variablestatements-in-catch-blocks" title></emu-xref>, then
+                        1. If _thisEnv_ is not the Environment Record for a |Catch| clause, let _bindingExists_ be *true*.
+                      1. Else,
+                        1. Let _bindingExists_ be *true*.
```

Comment:
```suggestion
                        1. If _thisEnv_ is not the Environment Record for a |Catch| clause, set _bindingExists_ to *true*.
                      1. Else,
                        1. Set _bindingExists_ to *true*.
```

Reply:
Fixed those and a couple more.

(Was ecmarkup complaining about these? If so, I didn't find it.)